students/fill0s0fear/hw-1/first.js

```diff
@@ -0,0 +1,25 @@
+let str = prompt('Enter a text');
+let maxRow = {
+  count: 0,
+  name: ''
+};
+let currentCount = 0;
+let currentNumber = str[0];
```

Comment:
Why do you use `let` here?

Reply:
the same as the first comment.

First comment:
I use let here because it's better then var.
Currently, I understood that const will be better. Using const allow us to not change incoming text somehow

Reply:
let better then var. Var outdated syntax